modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -169,6 +201,20 @@ calcOpticalFlowPyrLK(const GArray<GMat>    &prevPyr,
                            int              flags        = 0,
                            double           minEigThresh = 1e-4);
 
+/** @brief Gaussian Mixture based Background/Foreground Segmentation.
+The class generates a foreground mask.
+
+Output image is foreground mask, i.e. 8-bit unsigned 1-channel (binary) matrix @ref CV_8UC1.
+
```

Comment:
It may be great to describe how this stateful kernel works - that it preserves initialized `cv::createBackgroundSubtractorMOG2` object - and give details on where to put initializing parameters for the Subtractor. I mean, that we pass them via `compile_args`

Reply:
I need to discuss

Reply:
I think it's not the high-priority issue, we can add the description later. Please, keep this comment somewhere